libcob/coblocal.h

```diff
@@ -1,6 +1,6 @@
 /*
-   Copyright (C) 2007-2012, 2014-2022 Free Software Foundation, Inc.
-   Written by Roger While, Simon Sobisch, Ron Norman
+   Copyright (C) 2007-2012, 2014-2024 Free Software Foundation, Inc.
```

Comment:
we don't just include year numbers, so until @ddeclerck has reached a state where he considers to have many important commit merges done (or you miss a specific 3.x commit that he already merged) and we rebase the [java-interop](https://github.com/OCamlPro/gnucobol/tree/java-interop) branch which likely will add 2023 commits, the change should say ,2024`.